I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 140, in script
    $jones=DBTest()
  File "game/script.rpy", line 140, in <module>
    $jones=DBTest()
  File "game/script.rpy", line 21, in DBTest
    db.create('name', 'age', 'size')
IOError: [Errno 30] Read-only file system: u'../../dummy.pdl'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 140, in script
    $jones=DBTest()
  File "/Applications/renpy-7.3.5-sdk/renpy/ast.py", line 914, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "/Applications/renpy-7.3.5-sdk/renpy/python.py", line 2028, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 140, in <module>
    $jones=DBTest()
  File "game/script.rpy", line 21, in DBTest
    db.create('name', 'age', 'size')
  File "pydblite/pydblite.py", line 189, in create
  File "pydblite/pydblite.py", line 260, in commit
IOError: [Errno 30] Read-only file system: u'../../dummy.pdl'

Darwin-19.6.0-x86_64-i386-64bit
Ren'Py 7.3.5.606
Gates of Galloo 1.0
Sun Nov 22 22:51:25 2020
